lblod/frontend-lokaal-mandatenbeheer
# LMB-616 | Update person fractie relation when needed

## Description

Fractie on a person needs to be updated when a change in mandataris or fractie is done. Update it on all the possible places.

## How to test

1. Create a new person
2. Add a new mandataris to it in the current bestuursperiode (2019 - 2025)  => fractie another than onafhankelijk
3. Add a new mandataris to it in another bestuursperiode (2012 - 2019)  => fractie another than onafhankelijk
4. When going to the mandaten of that  person a blue button to become onafhankelijk should be visible 
5. When you press that button both the mandatarissen should go to onafhankelijke fractie
   5.1 The onafhankelijke fractie should have the same uri (check by changing naam to uri in history-rows.hbs)
6. When looking at the fractie value on your person it should say the same URI
7. When wijziginging of mandataris fractie  in the persons currentFractie schould be updated (current bestuursperiode)
8. When updating the fractie of  the second madnataris from another bestuursperiode and looking back at the person it should  still have the fractie URI of the madntaris in current bestuursperiode and not the URI that you just changed it to.
9. All of this done should solve the issue
 
## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/28 **mandataris service**
